packages/rum-core/test/emulate/mockPerformanceObserver.ts

```diff
@@ -16,9 +19,12 @@ export function mockPerformanceObserver() {
       disconnect() {
         instances.delete(instance)
       },
-      observe({ entryTypes, type }: PerformanceObserverInit) {
+      observe({ entryTypes, type, buffered }: PerformanceObserverInit) {
         instance.entryTypes = entryTypes || (type ? [type] : [])
         instances.add(instance)
+        if (buffered) {
+          notify(instance, bufferedEntries)
+        }
```

Comment:
**💬 suggestion:** ‏this is Safari behavior, but I'm not sure this is the standard behavior. We should at least state this as a comment. Maybe this could be done more explicitly by passing a `{ synchronousBufferedEntriesCallback: true }` flag to `mockPerformanceObserver`.

Reply:
Hum, the mock always triggers entries synchronously even for non buffered ones. To me, it's more by convenience to avoid playing with the clock each time we use `mockPerformanceObserver`. 
So I'm not sure it's a good idea to add a special behaviour for buffered entries and if we generalise it for any entries adding `synchronousBufferedEntriesCallback: true` everywhere will be a bit cumbersome. Wdyt?

Reply:
Let's not talk about async/sync, but whether the PerformanceObserver callback is called during or after `observe()`. My point is not to make everything asynchronous, just to make the `observe` default behavior closer to the standard.

Again, I'd be fine with just documenting this with a comment so we know why it's implemented like this.